metricbeat/module/system/core/config.go

```diff
@@ -0,0 +1,46 @@
+package core
+
+import (
+	"strings"
+
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/pkg/errors"
+)
+
+// Core metric types.
+const (
+	percentages = "percentages"
+	ticks       = "ticks"
+)
+
+// Config for the system core metricset.
+type Config struct {
+	Metrics  []string `config:"core.metrics"`
+	CPUTicks *bool    `config:"cpu_ticks"` // Deprecated.
```

Comment:
I meant 5.6. But possibly I misunderstood the original suggestion. You were suggesting adding the deprecation warnings in 5.6 and not having a migration path in 5.6 (e.g. we are telling the user not to use `cpu_ticks`, but not offering them an alternative in 5.6)?